.github/workflows/ci.yaml

```diff
@@ -67,6 +68,25 @@ jobs:
           )"
           [[ "${output}" == "${expected}" ]] || \
             (echo >&2 "Ouptput from integration test does not match:" "${output}"; exit 1)
+      - name: Execute in directory without shell.nix
+        uses: ./
+        with:
+          verbose: true
+          derivation-path: ./tests/integration_tests/shell.nix
```

Comment:
I'm a bit confused, since the name suggests that `shell.nix` is not there, but here you have the path to an existing `shell.nix` file. Is the point here that `./tests/integration_tests/shell.nix` is a path to a file yet a path to a directory is expected and so it is not supposed to work? If so, maybe some non-existing path would be a better example value here?

Reply:
The `derivation-path` supports a path to a directory that contains a `shell.nix`/`default.nix` or it supports a path to a specific `shell.nix`/`default.nix`.  If you specify a directory, the script changes into that directory to execute the `nix-shell` command. If you specify a path to a Nix file, the script changes into the directory that contains the Nix file and then passes the basename as the `path` for the `nix-shell` invocation. BTW, we change into the directory so that relative paths in the derivation evaluate properly.

I added some comments to the integration tests workflow to clarify what we are demonstrating. Does that make it clearer?

Reply:
Yeah, now it makes sense :+1: